R/transform.R

```diff
@@ -4,11 +4,17 @@
 #'   the result.
 #' @param files A character vector with paths to the file that should be
 #'   transformed.
+#' @param assert_code Boolean to indicate whether checking for parsing errors
+#'   should be done prior to styling.
 #' @inheritParams make_transformer
 #' @return A logical value that indicates whether or not any file was changed is
 #'   returned invisibly. If files were changed, the user is informed to
 #'   carefully inspect the changes via a message sent to the console.
-transform_files <- function(files, transformers) {
+transform_files <- function(files, transformers, assert_code = TRUE) {
```

Comment:
Ok, so we can remove the argument then. I am not sure whether it makes sense to implement the error handling in `transform_lines_enc()` since it has already a error handling but I don't know why it does only give a warning instead of an error.
``` r
library("styler")
style_text("a 3")
#> Error in parse(text = text, keep.source = TRUE): <text>:1:3: unexpected numeric constant
#> 1: a 3
#>       ^
path <- tempfile(fileext = ".R")
writeLines("a 3", path)
style_file(path)
#> Warning: When processing file6b1f0f56be.R: <text>:1:3: unexpected numeric constant
#> 1: a 3
#>       ^
#> Error in if (any(changed)) {: missing value where TRUE/FALSE needed
style_dir(dirname(path))
#> Warning: When processing ./file6b1f0f56be.R: <text>:1:3: unexpected numeric constant
#> 1: a 3
#>       ^
#> Error in if (any(changed)) {: missing value where TRUE/FALSE needed
```